src/components/Map.vue

```diff
@@ -149,16 +155,29 @@ export default {
           buildTileUrlTemplate: this.buildTileUrlTemplate,
           crossOrigin: this.crossOriginMedia
         };
+
+        if (this.stac.type === 'Collection' && data.type === 'FeatureCollection') {
+          data = this.stac;
+          options.fillOpacity = 0;
+          this.collectionsLayer = await stacLayer(this.stacLayerData, {
+            fillOpacity: 0,
+            weight: 2,
+            color: '#188191',
+            displayPreview: this.stacLayerData.features.length < 50
+          });
+          this.collectionsLayer.addTo(this.map);
+        }
+
         if (this.stac instanceof STAC) {
           options.baseUrl = this.stac.getAbsoluteUrl();
         }
         if ('href' in data) {
           if (this.stac.type === 'Feature') {
             options.bbox = this.stac?.bbox;
           }
-          else if (this.stac.type === 'Collection') {
-            options.bbox = this.stac?.extent?.spatial?.bbox[0];
-          }
+          // else if (this.stac.type === 'Collection') {
```

Comment:
I can't remember what this code block was for. If it is not needed you can simply remove it though.

Reply:
Yeah it seemed redundant - I'll double check what happens for a particular asset.